plugins/generic/pln/classes/form/PLNSettingsForm.inc.php

```diff
@@ -74,7 +74,19 @@ function readInputData() {
 	 * @see Form::display()
 	 */
 	function display() {
+		$journal =& Request::getJournal();
+		$issn = '';
+		if ($journal->getSetting('onlineIssn')) {
+			$issn = $journal->getSetting('onlineIssn');
+		} else if ($journal->getSetting('printIssn')) {
+			$issn = $journal->getSetting('printIssn');
+		}
+		$has_issn = false;
```

Comment:
Just a stylistic thing -- and I see it's probably elsewhere in this plugin -- but we generally use lowerCamelCase in variable names.